packages/web-react/src/components/Tabs/__tests__/useTabsStyleProps.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { renderHook } from '@testing-library/react';
+import React from 'react';
+import { BreakpointToken, ChildrenProps, SpaceToken } from '../../../types';
+import { TabsProvider } from '../TabContext';
+import { useTabsStyleProps } from '../useTabsStyleProps';
+
+const dataProvider: {
+  spacing: SpaceToken | Partial<Record<BreakpointToken, SpaceToken>> | undefined;
+  expectedStyle: Record<string, string | undefined>;
+}[] = [
+  { spacing: undefined, expectedStyle: {} },
+  {
+    spacing: 'space-100',
+    expectedStyle: { '--tabs-spacing': 'var(--spirit-space-100)' },
+  },
+  {
+    spacing: { tablet: 'space-100' },
+    expectedStyle: { '--tabs-spacing-tablet': 'var(--spirit-space-100)' },
+  },
+  {
+    spacing: { mobile: 'space-100', tablet: 'space-200', desktop: 'space-300' },
+    expectedStyle: {
+      '--tabs-spacing': 'var(--spirit-space-100)',
+      '--tabs-spacing-tablet': 'var(--spirit-space-200)',
+      '--tabs-spacing-desktop': 'var(--spirit-space-300)',
+    },
+  },
+];
```

Comment:
When I was working on the Link component, I noticed that this part of the tests was in separate files. You could separate them as well.

Reply:
I honestly don't find it very useful to separate it